src/BaGet/appsettings.json

```diff
@@ -22,6 +22,10 @@
     "PackageSource": "https://api.nuget.org/v3/index.json"
   },
 
+  "IISServerOptions": {
+    "MaxRequestBodySize": 262144000
+  },
```

Comment:
If you go with my comment above https://github.com/loic-sharma/BaGet/pull/473/files#r378645236, please remove these lines since they are no longer necessary for the 250MB default.